demos/object_detection_demo/cpp/proposal_common.hpp

```diff
@@ -0,0 +1,99 @@
+#include <models/model_base.h>
```

Comment:
please follow openvino convention and put system headers first, then ov header, then sample headers

Reply:
I see it differently:
1. Samples have `clang-format off` which is weird and suggest there is something wrong about includes in samples.
2. Putting sample header first verifies that the header is independent. Such header doesn't require corresponding .cpp file to include something else before that .hpp
3. Can you point me where this convention described?

I suspect @ilya-lavrenov may have an explanation. Can you comment?

Reply:
`clang-format off` is used to avoid  clang-format alphabet sorting of headers while we want to apply some logical structuring on them.

Reply:
This all is strange:
`SortIncludes` and `IncludeBlocks` are from https://clang.llvm.org/docs/ClangFormatStyleOptions.html
1. To suppress header sorting there is SortIncludes, no need to use that general `clang-format off`
2. IncludeBlocks option is already aware of blocks

Reply:
agree with @vladimir-dudnik